app/views/course/assessment/submission/submissions/_submissions.html.slim

```diff
@@ -0,0 +1,12 @@
+table.table.submissions
+  thead
+    tr
+      th = t('.student_name')
+      th = t('.status')
+      th = t('.grade')
+      th = t('.experience_points')
+      th
+  tbody
+    - course_users.each do |course_user|
```

Comment:
@lowjoel I did build the hash with course_user as key, and do the checks here. 

Didn't do the course_user_submission partial here cause I thought it might have been a little too many partials to deal with. 

Will this work? If you prefer having that partial (or a helper method) I can do a PR, it does help readability but not by much.